pgwidget/pgwidget_core.py

```diff
@@ -1613,6 +1605,8 @@ def __init__(self,pos,size,text,function=lambda *args:None,function_args=None,bo
         self.visibility_layer=100
         self.hover_color=hover_color
         self.hover_label_color=hover_label_color
+        self.fix_horizontal_position = fix_horizontal_position
+        self.fix_vertical_position = fix_vertical_position
```

Comment:
are these variables used somewhere?

Reply:
button.fix_horizontal_position + fix_vertical_position is not used anywhere in pgwidget

i would rather see something like 
```
def relocate_element_with_label(self, screen_size_offset, in_x_direction=None, in_y_direction=None)
    if in_x_direction is None:
        in_x_direction = not self.fix_horizontal_position\
...
```

Reply:
@YoungMasterGandalf - These two new attributes seem to me invalid and should be rather in layout object than in the button definition - requires deeper discussion